src/components/Table/index.tsx

```diff
@@ -76,26 +77,45 @@ export const Table = ({ headData, bodyData, isClosed }: TableProps) => {
         >
           {bodyData.map(({ purchases, color, title }) =>
             purchases.map((purchase: any, index) => {
+              console.log(purchase)
               return (
                 <TableRow key={index} padding={isMobile ? '0 0 0 10px' : '0 0 0 20px'}>
                   <TableColumn>
                     <TokenPriceLabel color={color}>{title}</TokenPriceLabel>
                   </TableColumn>
-                  {Object.getOwnPropertyNames(purchase).map((element: any, _index) => {
-                    return (
-                      <TableColumn key={_index}>
-                        <TokenPriceLabel>{purchase[element]}</TokenPriceLabel>
-                      </TableColumn>
-                    )
-                  })}
+                  {Object.getOwnPropertyNames(purchase)
+                    .filter(skip => skip !== 'status')
+                    .map((element: any, _index) => {
+                      return (
+                        <TableColumn key={_index}>
+                          <TokenPriceLabel>{purchase[element]}</TokenPriceLabel>
```

Comment:
Avoid `any` and cast the type using `keyof`

```suggestion
                          <TokenPriceLabel>{purchase[element as keyof purchase]}</TokenPriceLabel>
```